MetasysServices/NetworkDevices/NetworkDeviceServiceProvider.cs

```diff
@@ -58,7 +58,17 @@ public async Task<IEnumerable<MetasysObject>> GetAsync(string type = null)
             var response = await this.GetAllAvailablePagesAsync("networkDevices", new Dictionary<string, string> { { "type", type } }).ConfigureAwait(false);
             return ToMetasysObject(response, Version, MetasysObjectTypeEnum.Object);
         }
-
+        /// <inheritdoc/>
+        public IEnumerable<MetasysObject> GetByClassification(string classification = null)
+        {
+            return GetByClassificationAsync(classification).GetAwaiter().GetResult();
```

Comment:
try to use sync method of `GetAllAvailablePagesAsync()` if the synchronous method is not available, then we can remove this method as we don't want to use GetAwaiter().GetResult() to make the 'async' method 'sync'.

we can remove the 'sync' variation of this method and allow the client/consumers to use only the 'async' method.

Just for reference, https://github.jci.com/PuneWorkflow/metasys-scan-engine/pull/49#issuecomment-214703 from Carol, that says we should not use as well as not create methods that are using .GetResult(), .wait().